<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemsView</class>
 <widget class="QWidget" name="ItemsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>701</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitterInfo">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="widgetBrowser" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string extracomment="market_searchfield">&amp;Filter:</string>
           </property>
           <property name="buddy">
            <cstring>textFilter</cstring>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>8</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="textFilter">
           <property name="focusPolicy">
            <enum>Qt::ClickFocus</enum>
           </property>
           <property name="role" stdset="0">
            <string notr="true">frame</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonClearFilter">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string>Clea&amp;r</string>
           </property>
           <property name="role" stdset="0">
            <string notr="true">dialog-button</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkHidePrototypes">
           <property name="text">
            <string>Hide prototypes</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkLogicalOrder">
           <property name="text">
            <string>Logical order</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>5</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitterItems">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>5</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QTreeView" name="treeGroups">
             <property name="contextMenuPolicy">
              <enum>Qt::ActionsContextMenu</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideNone</enum>
             </property>
             <property name="uniformRowHeights">
              <bool>true</bool>
             </property>
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <property name="role" stdset="0">
              <string notr="true">tree</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <item>
            <widget class="QListView" name="listItems">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <property name="textElideMode">
              <enum>Qt::ElideNone</enum>
             </property>
             <property name="uniformItemSizes">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QStackedWidget" name="stackInfo">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pageNoItems">
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <property name="margin">
         <number>10</number>
        </property>
        <item>
         <widget class="QLabel" name="labelNoItems">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string extracomment="market_itemnotfound">No items found</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageInfo">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="labelName">
            <property name="role" stdset="0">
             <string notr="true">header</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QLabel" name="labelIcon">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>120</height>
               </size>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="textDescription">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>120</height>
               </size>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="tabParent" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="role" stdset="0">
           <string notr="true">tab-parent</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="role" stdset="0">
              <string notr="true">tree</string>
             </property>
             <widget class="QWidget" name="tabParameters">
              <attribute name="title">
               <string extracomment="entityinfo_parameters">&amp;PARAMETERS</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_11">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="GridTreeView" name="treeParameters">
                 <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                 </property>
                 <property name="textElideMode">
                  <enum>Qt::ElideNone</enum>
                 </property>
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="indentation">
                  <number>0</number>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <property name="uniformRowHeights">
                  <bool>true</bool>
                 </property>
                 <property name="itemsExpandable">
                  <bool>false</bool>
                 </property>
                 <property name="allColumnsShowFocus">
                  <bool>true</bool>
                 </property>
                 <attribute name="headerStretchLastSection">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabComponents">
              <attribute name="title">
               <string extracomment="entityinfo_components">&amp;COMPONENTS</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_12">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="QWidget" name="widgetDecodingLevel" native="true">
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>10</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>4</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="maximumSize">
                     <size>
                      <width>200</width>
                      <height>16777215</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>Decoding level</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                    <property name="role" stdset="0">
                     <string notr="true">header</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame">
                    <property name="frameShape">
                     <enum>QFrame::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <property name="spacing">
                      <number>2</number>
                     </property>
                     <property name="leftMargin">
                      <number>3</number>
                     </property>
                     <property name="topMargin">
                      <number>3</number>
                     </property>
                     <property name="rightMargin">
                      <number>3</number>
                     </property>
                     <property name="bottomMargin">
                      <number>4</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_4">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="pixmap">
                        <pixmap resource="../res/resources.qrc">:/decoder.png</pixmap>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QLabel" name="labelDecoderLevel"/>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="GridTreeView" name="treeComponents">
                 <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                 </property>
                 <property name="textElideMode">
                  <enum>Qt::ElideNone</enum>
                 </property>
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <property name="uniformRowHeights">
                  <bool>true</bool>
                 </property>
                 <property name="itemsExpandable">
                  <bool>false</bool>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="allColumnsShowFocus">
                  <bool>true</bool>
                 </property>
                 <property name="sortSequence" stdset="0">
                  <number>2</number>
                 </property>
                 <attribute name="headerStretchLastSection">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabComponentUse">
              <attribute name="title">
               <string>COMPONENT &amp;IN</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="GridTreeView" name="treeComponentUse">
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <property name="uniformRowHeights">
                  <bool>true</bool>
                 </property>
                 <property name="itemsExpandable">
                  <bool>false</bool>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="allColumnsShowFocus">
                  <bool>true</bool>
                 </property>
                 <property name="sortSequence" stdset="0">
                  <number>2</number>
                 </property>
                 <attribute name="headerStretchLastSection">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabRequirements">
              <attribute name="title">
               <string extracomment="entityinfo_extensions">E&amp;XTENSIONS</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_13">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="QTreeView" name="treeRequirements">
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="uniformRowHeights">
                  <bool>true</bool>
                 </property>
                 <property name="allColumnsShowFocus">
                  <bool>true</bool>
                 </property>
                 <property name="role" stdset="0">
                  <string notr="true">tree</string>
                 </property>
                 <attribute name="headerVisible">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabBonuses">
              <attribute name="title">
               <string extracomment="entityinfo_bonuses">&amp;BONUSES</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>4</number>
               </property>
               <property name="rightMargin">
                <number>4</number>
               </property>
               <property name="bottomMargin">
                <number>4</number>
               </property>
               <item>
                <widget class="GridTreeView" name="treeBonuses">
                 <property name="contextMenuPolicy">
                  <enum>Qt::CustomContextMenu</enum>
                 </property>
                 <property name="verticalScrollMode">
                  <enum>QAbstractItemView::ScrollPerPixel</enum>
                 </property>
                 <property name="rootIsDecorated">
                  <bool>false</bool>
                 </property>
                 <property name="uniformRowHeights">
                  <bool>true</bool>
                 </property>
                 <property name="itemsExpandable">
                  <bool>false</bool>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <property name="allColumnsShowFocus">
                  <bool>true</bool>
                 </property>
                 <property name="sortSequence" stdset="0">
                  <number>2</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionGroupsExpandAll">
   <property name="text">
    <string>Expand All</string>
   </property>
   <property name="shortcut">
    <string>X</string>
   </property>
  </action>
  <action name="actionGroupsCollapseAll">
   <property name="text">
    <string>Collapse All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionSortRow">
   <property name="text">
    <string>Sort Row</string>
   </property>
  </action>
  <action name="actionSortColumn">
   <property name="text">
    <string>Sort Column</string>
   </property>
  </action>
  <action name="actionCopyToClipboard">
   <property name="text">
    <string>Copy To Clipboard</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionSetReference">
   <property name="text">
    <string>Set Reference</string>
   </property>
  </action>
  <action name="actionInformation">
   <property name="text">
    <string>&amp;Information</string>
   </property>
   <property name="shortcut">
    <string>I</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GridTreeView</class>
   <extends>QTreeView</extends>
   <header>gridtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonClearFilter</sender>
   <signal>clicked()</signal>
   <receiver>textFilter</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGroupsCollapseAll</sender>
   <signal>triggered()</signal>
   <receiver>treeGroups</receiver>
   <slot>collapseAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGroupsExpandAll</sender>
   <signal>triggered()</signal>
   <receiver>treeGroups</receiver>
   <slot>expandAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
